test/Makefile.am.inc

```diff
@@ -79,24 +79,20 @@ AM_JS_LOG_FLAGS = \
 	@JASMINE_REPORT_ARGUMENT@ \
 	--no-config \
 	$(NULL)
-LOG_COMPILER = $(top_srcdir)/test/gtester-mock-event-recorder.sh
-AM_LOG_FLAGS = -k --verbose
+LOG_COMPILER = gtester
+AM_LOG_FLAGS = -k --tap --verbose
```

Comment:
You're right; currently, the test suite is completely failing for me in a jhbuild environment as well.

99% of this build system and code assumes it's running on a Debian/Ubuntu machine... :-(